.rubocop.yml

```diff
@@ -10,7 +10,7 @@ AllCops:
   NewCops: disable
   SuggestExtensions: false
   Exclude:
-    - "test/dummy/db/**/*.rb"
+    - "test/dummy/**/*"
```

Comment:
We shouldn't care about styling in the dummy app.

Reply:
Why not?